### PR TITLE
Fixing typos

### PR DESCRIPTION
**Pull request by ID6109**
## Who can review?

@LukeWood @ianstenbit @jbischof @haifeng-jin

### CODE REVIEW

**Review by jbischof — APPROVED** [review]
Very nice @ID6109, did you use a tool for this?

**Comment by ID6109** [comment]
> Very nice @ID6109, did you use a tool for this?

The IDE I use has built-in functionality for checking typos and basic spelling errors. From there, it is a manual iterative process of going through all the "obscure" words (in our case, ~700 in total).

File: keras_cv/point_cloud/point_cloud.py [hunk 1]
```diff
@@ -105,7 +105,7 @@ def group_per_sample(box_index):
 # TODO(lengzhaoqi/tanzhenyu): compare the performance with v1
 def is_within_any_box3d_v2(points, boxes, keepdims=False):
     """Checks if 3d points are within 3d bounding boxes.
-    Currently only xyz format is supported.
+    Currently, only xyz format is supported.
```
[the remark this answers is not in the document]
Comment (ID6109) [reply keras_cv/point_cloud/point_cloud.py]: Reverted

**Pull request merged**